.github/workflows/ci.yaml

```diff
@@ -0,0 +1,321 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  merge_group:
+  pull_request:
+    branches:
+      - "main"
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+"
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-20.04 
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.9.1
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Check out code
+      uses: actions/checkout@v2
```

Comment:
deprecated, use v4

Reply:
OK, I updated to be **v4** in **lint, unit, metallb_e2e** but in **e2e, olm** is still **v2**.
About **upgrade_version**: the name of the steps- Checkout Previous Release Metal LB Operator and Checkout Latest Metal LB Operator is **v2**
and the name of the step : Checkout is **V4**.
@oribon